src/app/routes/api/v3/admin/forms/__tests__/admin-forms.form.routes.spec.ts

```diff
@@ -2126,4 +2127,39 @@ describe('admin-form.form.routes', () => {
       expect(resp.body).toEqual(expectedResponse)
     })
   })
+
+  describe('PUT /admin/forms/:formId/end-page', () => {
+    const MOCK_END_PAGE: Partial<FormEndPage> = {
+      title: 'end page title',
+      buttonText: 'end page button',
+    }
+
+    it('should return 400 when button link does not use valid URI scheme', async () => {
```

Comment:
can we have a positive test for this route too?

Reply:
added!